src/modules/Crowbar.rb

```diff
@@ -45,7 +46,8 @@ def main
       # Path to the files with JSON data
       @network_file = Installation.destdir + "/etc/crowbar/network.json"
       @crowbar_file = Installation.destdir + "/etc/crowbar/crowbar.json"
-      @installed_file = "/opt/dell/crowbar_framework/.crowbar-installed-ok"
+      @crowbarrc_file = Installation.destdir + "/etc/crowbarrc"
```

Comment:
```suggestion
      @crowbarrc_file = File.join(Installation.destdir, "/etc/crowbarrc")
```

note: it's better to use `File.join` instead of plain `+`, that better handles adding the path separator.

Reply:
I was being consistent, but you're right. I fixed all of these.